spec/turn_spec.rb

```diff
@@ -93,7 +93,7 @@
     end
 
     context "duplicate color guesses" do
-      it "returns two :exact, ignoring two partials" do
+      fit "returns two :exact, ignoring two partials" do
```

Comment:
Revert this, please, so other tests also run 🙂

Reply:
Done! Apology for repeated mistakes.